forge-gui-desktop/src/test/java/forge/deck/DeckRecognizerTest.java

```diff
@@ -44,69 +44,67 @@ private void initMaps(){
         }
     }
 
-    /* ======================================
-     * TEST SINGLE (Card and Non-Card) Parts
-     * - CardName
-     * - Card Amount
-     * - Set Code
-     * - Collector Number
-     * - Deck Name
-     * - Card Type
-     * - Deck Section
-     * - Mana Colour
-     * ======================================
+    /*
+     * ====================================== TEST SINGLE (Card and Non-Card) Parts
+     * - CardName - Card Amount - Set Code - Collector Number - Deck Name - Card
+     * Type - Deck Section - Mana Colour ======================================
```

Comment:
i think these doc lines shouldn't be compressed?

Reply:
Agreed

Reply:
Could you please point me to a correct/acceptable example?
This PR is mostly a succession of automated refactorings...

Reply:
@Hanmac BTW: Thanks for your detailed review!

Reply:
before the Doc Lines showed some Kind of list, now after the auto refactor it doesn't anymore

Reply:
Thanks. Fixed.